elide-integration-tests/src/test/java/com/yahoo/elide/tests/AnyPolymorphismIT.java

```diff
@@ -216,41 +195,57 @@ public void testAnySubpaths() {
                 .given()
                 .contentType(JSONAPI_CONTENT_TYPE)
                 .accept(JSONAPI_CONTENT_TYPE)
-                .body(jsonParser.getJson(tractorAsPropertyFile))
-                .post(propertyPath)
+                .body(jsonParser.getJson("/AnyPolymorphismIT/AddTractorProperty.json"))
+                .post("/property")
                 .then()
                 .assertThat()
                 .statusCode(HttpStatus.SC_CREATED)
                 .extract()
-                .path(idPath);
+                .path("data.id");
 
-        RestAssured
+        Response response = RestAssured
                 .given()
                 .contentType(JSONAPI_CONTENT_TYPE)
                 .accept(JSONAPI_CONTENT_TYPE)
-                .get(propertyPath + "?page[totals]")
+                .get("/property?page[totals]")
                 .then()
                 .assertThat()
                 .statusCode(HttpStatus.SC_OK)
-                .body("meta.page.totalRecords", greaterThan(0));
+                .extract()
+                .response();
+
+        Integer collectionSize = response.path("data.size()");
+        Integer totalSize = response.path("meta.page.totalRecords");
+        Assert.assertEquals(collectionSize, totalSize);
 
         RestAssured
                 .given()
                 .contentType(JSONAPI_CONTENT_TYPE)
                 .accept(JSONAPI_CONTENT_TYPE)
-                .get(propertyPath + sep + id + "/myStuff")
+                .get("/property/" + id + "/myStuff")
                 .then()
                 .assertThat()
                 .statusCode(HttpStatus.SC_OK)
-                .body("data.attributes.horsepower", equalTo(horsepower),
-                        idPath, equalTo(one));
+                .body("data.attributes.horsepower", equalTo(102),
+                        "data.id", equalTo("1"));
+
+        //show that you can't get a relation by the interface type
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .get("/property/" + id + "/device")
```

Comment:
I really meant to test /device.

Since it is a mapped interface, I wonder how loading a root level collection for a mapped interface might behave.

Reply:
It 404s the exact same way as this path does.  Which is what I'd expect because the mappings belong to the entities. a MappedInterface isn't an entity.